addons/survey/views/survey_templates.xml

```diff
@@ -432,7 +432,7 @@
                 <t t-foreach='question.suggested_answer_ids' t-as='label'>
                     <t t-set="item_idx" t-value="label_index"/>
                     <t t-set="answer_selected" t-value="answer_line and answer_line.suggested_answer_id.id == label.id"/>
-                    <t t-set="is_correct" t-value="label.answer_score > 0.0"/>
+                    <t t-set="is_correct" t-value="label.is_correct"/>
```

Comment:
Commit guidelines: use a verb.
Suggestion: [FIX] survey: use bg-success only if answer is_correct

Reply:
You can also add task-... in the commit message.
In the message, you can also add that the change concerns MCQs.

Reply:
@Odoonan Done